acdc/TLACDCExperiment.py

```diff
@@ -98,6 +99,10 @@ def __init__(
         self.names_mode = names_mode
         self.use_pos_embed = use_pos_embed
         self.show_full_index = show_full_index
+        
+        self.positions = positions if positions is not None else [None]
+        if self.positions != [None]:
+            raise NotImplementedError("Splitting by position not implemented yet")
```

Comment:
This is the only change, and shouldn't matter because you're not using positions yet